samples/nrf9160/modem_shell/README.rst

```diff
@@ -0,0 +1,467 @@
+
+Open an IPv6 UDP socket::
+
+   sock connect -a 1a2b:1a2b:1a2b:1a2b::1 -p 20000 -f inet6 -t dgram
+
+Open a raw socket::
+
+   sock connect -f packet -t raw
+
+Open a socket to a non-default PDP context::
+
+   link connect -a nondefault.context.com
+   sock connect -f packet -t raw -I 1
+
+Send a string through the socket::
+
+   sock send -i 0 -d testing
+
+Send 100 kB of data and show throughput statistics::
+
+   sock send -i 0 -l 100000
+
+Send data periodically with 10 s interval::
+
+   sock send -i 0 -e 10 -d test_periodic
+
+Calculate the receive throughput::
+
+   <do whatever is needed to make device receive data after some time>
+   sock recv -i 0 -r -l 1000000
+   sock recv -i 0
+   sock recv -i 0
+
+Close a socket::
+
+   sock close -i 0
+
+Use RAI settings::
+
+   link funmode -4
+   sock rai --rai_enable
+   link funmode -1
+   sock connect -a 111.222.111.222 -p 20000
+   sock rai -i 0 --rai_last
+   sock send -i 0 -d testing
+
+List open sockets::
+
+   sock list
+
+SMS tool
+========
+
+MoSh command: "sms"
+
+You can use the SMS tool for sending and receiving SMS messages.
+
+Examples
+--------
+
+Register the SMS service so that messages can be received if SIM subscription supports it::
+
+   sms reg
+
+Send an SMS message (registration is done automatically if not already done)::
+
+   sms send -n +987654321 -m testing
+
+GNSS
+====
+
+MoSh command: "gnss"
+
+GNSS provides commands for searching the location of the device.
+
+Examples
+--------
+
+Start GNSS tracking and stop it::
+
+   gnss start
+   gnss stop
+
+Disable LTE, enable all GNSS output and start continuous tracking with power saving enabled::
+
+   link funmode --lteoff
+   gnss output 2 1 1
+   gnss mode cont
+   gnss config powersave perf
+   gnss start
+
+Enable LTE PSM, only NMEA output, automatic A-GPS data fetching and start periodic fixes with 5 minute interval and 120 second timeout::
+
+   link psm -e
+   gnss output 0 1 0
+   gnss agps automatic enable
+   gnss mode periodic 300 120
+   gnss start
+
+Enable Low Noise Amplifier (LNA) for use with the on-board GPS antenna::
+
+   gnss lna enable
+
+FOTA
+====
+
+MoSh command: "fota"
+
+You can use FOTA to perform software updates over-the-air for both modem and application side.
+However, to use this feature, you need to know which updates are available in the servers.
+This feature is intended to be used only by selected users and customers to whom available image names are communicated separately.
+
+Examples
+--------
+
+Perform a FOTA update::
+
+   fota download eu fw_update_package_filename.hex
+
+PPP
+===
+
+MoSh command: "ppp"
+
+You can use the PPP (Point-to-Point Protocol) to enable dial-up access to the Internet.
+The MoSh command is simple but you need to have a normal dial-up setup in your PC to be able to use the development kit's PPP interface.
+
+.. note::
+   Dial-up connection is non functional when using Segger virtual UART ports.
```

Comment:
@peknis Please check this note and especially the way I've added the reference here and into the References chapter.

Reply:
Leave a blank line after the ``.. note::``.
For the link, add a new line in the ..\..\ncs\nrf\doc\nrf.links.txt (in the Source: infocenter.nordicsemi.com section) like this:
``.. _`nRF9160 Hardware Verification Guidelines`: https://infocenter.nordicsemi.com/topic/nwp_034/WP/nwp_034/nwp_034_intro.html``

Then in this doc: ``Refer to Section 2.1.2 of the `nRF9160 Hardware Verification Guidelines`_``

Reply:
I've changed these and let's see how it looks in the final document

Reply:
Modified link based on Teams chat:
`.. _`nRF9160 Hardware Verification Guidelines - UART interface`: https://infocenter.nordicsemi.com/topic/nwp_034/WP/nwp_034/nwp_034_uart_if.html`